crates/ruff_linter/src/rules/ruff/snapshots/ruff_linter__rules__ruff__tests__RUF036_RUF036.py.snap

```diff
@@ -2,58 +2,246 @@
 source: crates/ruff_linter/src/rules/ruff/mod.rs
 snapshot_kind: text
 ---
-RUF036.py:4:16: RUF036 `None` not at the end of the type annotation.
+RUF036.py:4:16: RUF036 [*] `None` not at the end of the type annotation.
   |
 4 | def func1(arg: None | int): 
   |                ^^^^ RUF036
 5 |     ...
   |
+  = help: Move `None` to the end of the type annotation
 
-RUF036.py:8:16: RUF036 `None` not at the end of the type annotation.
+ℹ Safe fix
+1 1 | from typing import Union as U
+2 2 | 
+3 3 | 
+4   |-def func1(arg: None | int): 
+  4 |+def func1(arg: int | None): 
+5 5 |     ...
+6 6 | 
+7 7 | 
+
+RUF036.py:8:16: RUF036 [*] `None` not at the end of the type annotation.
   |
 8 | def func2() -> None | int: 
   |                ^^^^ RUF036
 9 |     ...
   |
+  = help: Move `None` to the end of the type annotation
+
+ℹ Safe fix
+5 5 |     ...
+6 6 | 
+7 7 | 
+8   |-def func2() -> None | int: 
+  8 |+def func2() -> int | None: 
+9 9 |     ...
+10 10 | 
+11 11 | 
 
 RUF036.py:12:16: RUF036 `None` not at the end of the type annotation.
    |
 12 | def func3(arg: None | None | int): 
    |                ^^^^ RUF036
 13 |     ...
    |
+   = help: Move `None` to the end of the type annotation
 
-RUF036.py:12:23: RUF036 `None` not at the end of the type annotation.
+RUF036.py:12:23: RUF036 [*] `None` not at the end of the type annotation.
    |
 12 | def func3(arg: None | None | int): 
    |                       ^^^^ RUF036
 13 |     ...
    |
+   = help: Move `None` to the end of the type annotation
+
+ℹ Safe fix
+9  9  |     ...
+10 10 | 
+11 11 | 
+12    |-def func3(arg: None | None | int): 
+   12 |+def func3(arg: int | None): 
```

Comment:
We should avoid removing duplicate `None` values. While the second `None` is useless, it's not what the rule is about